src/interfaces/Config.ts

```diff
@@ -51,10 +51,35 @@ export interface IConfigTaxonomyUNFCCC {
   _document: IConfigDocumentMetadata
 }
 
+export interface IConfigTaxonomyMCF {
```

Comment:
yep, I have added a type for every property across the MCFs metadata and where a property only exists on one for instance I have added allowed for an undefined entry i.e approved_ref only exists on GCf so its `approved_ref?: IConfigMeta` with the `?:` @katybaulch

Reply:
I think we're going to need to be more explicit considering how we are planning on dynamically rendering family metadata which is taken from the schema. Let's catch up later today/next week about how the dynamic rendering affects this :)

Reply:
Yes it would be good to catch up, you have a lot more context than me as I am not too clear on how dynamic rendering would affect the types that we define.